deploy/docs/Dev.md

```diff
@@ -0,0 +1,34 @@
+# Development
+
+This document contains information helpful for developers.
+
+## Installation of non-official Helm charts
+
+Non-official Helm charts are available in [dev](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/gh-pages/dev) directory on [gh-pages](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/gh-pages) branch.
```

Comment:
Fancy markdown tricks I learnt recently

```suggestion
Non-official Helm charts are available in [dev] directory on [gh-pages] branch.

[dev]: https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/gh-pages/dev
[gh-pages]: https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/gh-pages
```

Reply:
@pmalek-sumo shouldn't those links be stored at the end of the file? 🤔

Reply:
Matter of preference I guess 🤷🏻